internal/handler/msg_usersinfo.go

```diff
@@ -59,7 +59,7 @@ func (h *Handler) MsgUsersInfo(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 
 	common.Ignored(
 		document, h.L,
-		"showCredentials", "showCustomData", "showPrivileges",
```

Comment:
Do we need a test for this one being set as `true`/`false` explicitly to ensure that the response has all the necessary fields and only them? I see it's being used as `true` in the `TestUsersinfo` test, but the logic looks too complicated there. Is there a way to simplify it?

Also, in `supported-commands.md` we can mark it as supported.

Reply:
I've updated the documentation, simplified the tests a bit, and added some new test cases.